projects/1.19/assets/client-commands/clientcommand/lang/zh_cn.json

```
@@ -0,0 +1,311 @@
+  "commands.cfish.error.openWater.lilyPad": "提示：尝试使浮标卡在睡莲下",
+  "commands.cfish.error.outOfWater": "浮标不在水中，自动控制无法工作",
+  "commands.cfish.help.tooManyEnchants": "提示：把多个魔咒放在一起获得会很难且可能需要很长时间",
+  "commands.cfish.listGoals.noGoals": "没有钓鱼目标",
+  "commands.cfish.listGoals.success": "有%d个钓鱼目标：",
+  "commands.cfish.needFishingManipulation": "本命令需要启用钓鱼控制",
+  "commands.cfish.removeGoal.success": "成功移除目标%s",
+  "commands.cfish.wrongLoot": "校正%dms后未获得目标战利品，可能超前了%s刻",
+
+  "commands.cfov.success": "将视场角设为%f",
+
+  "commands.cfunction.limitReached": "已达到命令限制（%d）",
+  "commands.cfunction.success": "从函数%s中运行%d个命令",
+
+  "commands.cgamma.success": "将伽马值设为%f",
+
+  "commands.cghostblock.fill.success": "成功填充%d个幽灵方块",
+  "commands.cghostblock.set.success": "成功放置幽灵方块",
+
+  "commands.cgive.success": "成功给予自己%d个%s",
+  "commands.cgive.notCreative": "仅创造模式玩家可以将物品给予自己",
+
+  "commands.cglow.entity.keepSearching.success": "正在使实体发光",
+  "commands.cglow.entity.failed": "没有目标实体可以发光",
+  "commands.cglow.entity.success": "成功使%d个实体发光",
+  "commands.cglow.area.success": "成功使%d个区域发光",
+
+  "commands.chotbar.notCreative": "仅创造模式玩家可以恢复物品栏",
+  "commands.chotbar.restoredHotbar": "成功恢复物品栏%d",
+
+  "commands.citemgroup.notFound": "未找到创造模式物品栏“%s”",
+  "commands.citemgroup.outOfBounds": "索引%d越界",
+  "commands.citemgroup.saveFile.failed": "无法保存创造模式物品栏文件",
+  "commands.citemgroup.addGroup.success": "成功添加创造模式物品栏“%s”",
+  "commands.citemgroup.addGroup.illegalCharacter": "创造模式物品栏只能拥有匹配[a-z0-9/._-]的字符",
+  "commands.citemgroup.addGroup.alreadyExists": "创造模式物品栏“%s”已经存在",
+  "commands.citemgroup.removeGroup.success": "成功移除创造模式物品栏“%s”",
+  "commands.citemgroup.addStack.success": "成功将%s添加到“%s”",
+  "commands.citemgroup.removeStack.success": "成功将“%s”中第%d个物品移除",
+  "commands.citemgroup.setStack.success": "成功将“%s”中第%d个物品设为%s",
+  "commands.citemgroup.changeIcon.success": "成功将“%s”的图标由%s替换为%s",
+  "commands.citemgroup.renameGroup.success": "成功将“%s”重命名为“%s”",
+
+  "commands.ckit.notFound": "未找到背包存档“%s”",
+  "commands.ckit.saveFile.failed": "无法保存背包存档文件",
+  "commands.ckit.load.success": "成功将背包存档“%s”给予自己",
+  "commands.ckit.load.notCreative": "仅创造模式玩家可以给自己物品",
+  "commands.ckit.create.success": "成功创建背包存档“%s”",
+  "commands.ckit.create.alreadyExists": "背包存档“%s”已经存在",
+  "commands.ckit.delete.success": "成功删除背包存档“%s”",
+  "commands.ckit.edit.success": "成功修改背包存档“%s”",
+  "commands.ckit.list": "可用背包存档：%s",
+  "commands.ckit.list.empty": "无可用背包存档",
+
+  "commands.cplayerinfo.ioException": "出现错误",
+  "commands.cplayerinfo.getNameHistory.success": "%s有如下名称：%s",
+
+  "commands.cparticle.unsuitableParticleOption": "无法在视频设置中“粒子效果”一项为“最少”时播放粒子效果",
+  "commands.cparticle.success": "成功显示粒子效果%s",
+
+  "commands.cpermissionlevel.success": "你的权限等级为%d",
+
+  "commands.cplaysound.success": "成功播放音效%s给自己",
+
+  "commands.cping.success": "你的平均ping是%dms",
+  "commands.cping.success.other": "%s的平均ping是%dms",
+  "commands.cping.singleplayer": "你在一个单人世界里(╯°□°）╯︵ ┻━┻",
```

```suggestion
  "commands.cping.singleplayer": "你在一个单人世界里（╯°□°）╯︵ ┻━┻",
```
对应左右括号
虽然颜文字这种应该都随便的（）